src/kitovu/cli.py

```diff
@@ -24,6 +24,11 @@
 from kitovu.sync import syncing
 
 
+class CliReporter(utils.AbstractReporter):
```

Comment:
Add a blank line after this (to answer your question from the other PR: Yeah, after `class Foo:` there should always be a blank line (or a docstring and then a blank line).

Not sure why pylint/flake8 aren't catching this, I *think* they are in qutebrowser.